src/pages/overview/pdf-accessibility-auto-tag-api/quickstarts/dotnet/index.md

```diff
@@ -0,0 +1,333 @@
+
+![Sign in](./shot1.png)
+
+2) After registering or logging in, you will then be asked to name your new credentials. Use the name, "New Project". 
+
+3) Change the "Choose language" setting to ".Net". 
+
+4) Also note the checkbox by, "Create personalized code sample." This will include a large set of samples along with your credentials. These can be helpful for learning more later. 
+
+5) Click the checkbox saying you agree to the developer terms and then click "Create credentials."
+
+![Project setup](./shot2.png)
+
+6) After your credentials are created, they are automatically  downloaded:
+
+![alt](./shot3.png)
+
+## Step Two: Setting up the project
+
+1) In your Downloads folder, find the ZIP file with your credentials: PDFServicesSDK-.NetSamples.zip. If you unzip that archive, you will find a README file, your private key, and a folder of samples:
+
+![alt](./shot5.png)
+
+2) We need two things from this download. The `private.key` file (as shown in the screenshot above, and the `pdfservices-api-credentials.json` file. You can find this in the `adobe-DC.PDFServicesSDK.NET.Samples` folder, inside any of the sample subdirectories, so for example, the `CombinePDF` folder.
+
+![alt](./shot6.png)
+
+<InlineAlert slots="text" />
+
+Note that that private key is *also* found in this directory so feel free to copy them both from here.
+
+3) Take these two files and place them in a new directory.
+
+4) In your new directory, create a new file, `ExtractTextInfoFromPDF.csproj`. This file will declare our requirements as well as help define the application we're creating.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="log4net" Version="2.0.12" />
+        <PackageReference Include="Adobe.PDFServicesSDK" Version="2.2.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <None Update="pdfservices-api-credentials.json">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+        <None Update="private.key">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+        <None Update="extractPDFInput.pdf">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+        <None Update="log4net.config">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+    </ItemGroup>
+
+</Project>
+```
+
+Our application will take a PDF, `Adobe Extract API Sample.pdf` (downloadable from [here](/Adobe%20Extract%20API%20Sample.pdf)) and extract it's contents. The results will be saved as a ZIP file, `ExtractTextInfoFromPDF.zip`. We will then parse the results from the ZIP and print out the text of any `H1` headers found in the PDF.

```

Comment:
No it won't. This is the text from the Extract quickstart.

Reply:
+1, this is incorrect for Autotag section.